emuflight/EmuFlight
# ImmersionRC GHST protocol

This is port of https://github.com/betaflight/betaflight/pull/10274

Massive thanks to @tonycake and @mikeller for original work.

### Progress:
✅  Control link
✅ Telemetry (Requires latest nightly build of OpenTX for battery info to be shown in sensors)
✅  Telemetry Unittests
❌  Protocol Unittests

Looking for testers with Ghost modules.

### Testing:

1. Flash the build
2. Enable SerailRX on relevant uart port
3. set GHST protocol via CLI (not available in configurator yet)
`set serialrx_provider = GHST`

You should then see channels respond in reciever tab
